CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ There are several placeholder values in both file names and the files themselves
 - `MODULE_NAME` should be replaced with a display name for the module as this is visible to players such as `Bat Grenades` or `Enderman Support Class`. This should match the name on the website if the module already exists.
 
 ### Folders
-Within the `data` folder there are three folders for the different namespaces this module will use. If the module is an extension, you will need an additional module for the base module.
+Within the `data` folder there are three folders for the different namespaces this module will use. If the module is an extension, you will need an additional module for the base module. Because we merge modules with a copy of the base clock certain folders and files should not be included once the module is submitted but are included in the template so you can test your module.
```

Comment:
```suggestion
Within the `data` folder there are three folders for the different namespaces this module will use. If the module is an extension, you will need an additional module for the base module. Because we merge modules with a copy of the base clock, certain folders and files should not be included once the module is submitted but are included in the template so you can test your module.
```